src/App.jsx

```diff
@@ -1,45 +1,12 @@
 import React from 'react';
 import './App.scss';
-
-export const misha = {
-  name: 'Misha',
-  age: 37,
-  sex: 'm',
-  isMarried: true,
-  partnerName: 'Natasha',
-};
-
-export const olya = {
-  name: 'Olya',
-  sex: 'f',
-  isMarried: true,
-  partnerName: 'Maksym',
-};
-
-export const alex = {
-  name: 'Alex',
-  age: 25,
-  sex: 'm',
-  isMarried: false,
-};
+import { Person } from './components/Person/Person';
+import { people } from './components/People/People';
```

Comment:
It is definitely NOT a component :) 

If you want to separate people values from your component => you can create a simple helpers.js file near this component, so the structure will be:

components
   // => People
        // => People.js
        // => helpers.js
        // => index.js (imports People component and exports it: `export { default } from './People';`)